### PR TITLE
Can nav through main controllers

### PR DESCRIPTION
**Pull request by sam-kim**
Profile and sign in/sign out should be on right side of page.

signout does not work because /users/sign_out looks for a user with id=sign_out.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -48,37 +48,39 @@
         <a class="navbar-brand" href="#">Llamame</a>
         <div class="navbar-collapse collapse navbar-responsive-collapse">
           <ul class="nav navbar-nav">
-            <li><%= link_to "Link1", "/path1"  %></li>
-            <li><%= link_to "Link2", "/path2"  %></li>
-            <li><%= link_to "Link3", "/path3"  %></li>
+            <li><%= link_to "Profile", "/users/#{current_user.id}"  %></li>
+            <% if current_user %>
```
Comment (rposborne) [inline app/views/layouts/application.html.erb]: Lets make sure indentation stays consistent.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -48,37 +48,39 @@
...
+            <li><%= link_to "Sign In", "/users/sign_in"  %></li>
+            <% end %>
           </ul>
         </div>
       </div>
     </div>
 
     <div class="container">
       <div class="row">
-        <div class="col-lg-9">
```
Comment (rposborne) [inline app/views/layouts/application.html.erb]: Why was this removed?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -48,37 +48,39 @@
...
-          <%= bootstrap_flash %>
-          <%= yield %>
-        </div>
```
Comment (rposborne) [inline app/views/layouts/application.html.erb:64]: Why was this removed?

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -48,37 +48,39 @@
...
-              <li class="nav-header">Sidebar</li>
-              <li><%= link_to "Link1", "/path1"  %></li>
-              <li><%= link_to "Link2", "/path2"  %></li>
-              <li><%= link_to "Link3", "/path3"  %></li>
+              <li class="nav-header">Rename Here</li>
+              <li><%= link_to "Company", "/companies"  %></li>
+              <li><%= link_to "People", "/people"  %></li>
+              <li><%= link_to "Interactions", "/interactions"  %></li>
             </ul>
+            <%= yield :sidebar if content? %>
```
Comment (rposborne) [inline app/views/layouts/application.html.erb]: This should be something along the lines of `yield(:sidebar) if content_for?(:sidebar)`

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -48,37 +48,39 @@
...
           </div><!--/.well -->
         </div><!--/span-->
+        <div class="col-lg-9">
+          <%= bootstrap_flash %>
+          <%= yield %>
+        </div>
```
Comment (rposborne) [inline app/views/layouts/application.html.erb]: At the bottom?

**Comment by mikewesthub** [comment]
:shipit:
